### PR TITLE
Somewhat belated post about R Foundation code of conduct policy

### PR DESCRIPTION
**Pull request by karawoo**
Let me know if it needs any changes.

### CODE REVIEW

**Comment by rgaiacs** [comment]
@karawoo Thanks for the pull request. SCF's code of conduct is available at http://software-carpentry.org/conduct.html. Could you mention if SCF's code of conduct is (or not) compatible with the R Foundation's code of conduct?

File: blog/2015/11/r-code-of-conduct.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+layout: blog
+root: ../../..
+author: Kara Woo
+title: "R Foundation Announces Code of Conduct Policy"
+date: 2015-11-08
+time: "12:00:00"
+category: []
```
Comment (gvwilson) [inline blog/2015/11/r-code-of-conduct.html]: category: ["Community", "R Foundation"]

File: blog/2015/11/r-code-of-conduct.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+<p>
+  This announcement came as welcome news to members of the R community who had
+  requested such a policy. Last June many R users formalized this request in
+  an <a href="https://docs.google.com/document/d/1C1oPhup72lPHJXbpyJZNIo1BdCzfxZ_VoiWJWzmufjU/edit?usp=sharing">open
+  letter to the R Foundation</a>.
+</p>
+
+<p>
+  With its new policy, the R Foundation joins a growing number of academic and
+  professional organizations that have instituted codes of conduct for all of
```
Comment (gvwilson) [inline blog/2015/11/r-code-of-conduct.html]: professional organizations (including Software Carpentry)

and then make a hyperlink around 'Software Carpentry' to "{{page.root}}/conduct.html".

**Comment by karawoo** [comment]
Thanks for the feedback @rgaiacs and @gvwilson. I've made some edits. I tried to clarify that the R Foundation isn't imposing one specific code of conduct, so compatibility isn't an issue.

**Comment by gvwilson** [comment]
:+1:

**Pull request merged**